internal/test_cases/exit_test_case.go

```diff
@@ -0,0 +1,77 @@
+package test_cases
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/codecrafters-io/shell-tester/internal/condition_reader"
+	"github.com/codecrafters-io/shell-tester/internal/logged_shell_asserter"
+	"github.com/codecrafters-io/shell-tester/internal/shell_executable"
+	"github.com/codecrafters-io/shell-tester/internal/utils"
+	virtual_terminal "github.com/codecrafters-io/shell-tester/internal/vt"
+	"github.com/codecrafters-io/tester-utils/logger"
+)
+
+// ExitTestCase is a test case that:
+// Sends an exit command to the shell
+// Verifies that the shell exits with the expected exit code
+// If any error occurs returns the error from the corresponding assertion
+type ExitTestCase struct {
+	// Command is the exit command to send to the shell (e.g. "exit 0")
+	Command string
+
+	// ExpectedExitCode is the expected exit code
+	ExpectedExitCode int
+}
+
+func (t ExitTestCase) Run(asserter *logged_shell_asserter.LoggedShellAsserter, shell *shell_executable.ShellExecutable, logger *logger.Logger, skipSuccessMessage bool) error {
```

Comment:
```suggestion
func (t ExitTestCase) Run(asserter *logged_shell_asserter.LoggedShellAsserter, shell *shell_executable.ShellExecutable, logger *logger.Logger) error {
```

I don't see `skipSuccessMessage` being used anywhere so let's remove - if needed, we'll bring it back but keep in the `ExitTestCase` struct as `ShouldSkipSuccessMessage`, not as a function param